com.adsbynimbus.nimbus/Runtime/Scripts/Nimbus.Internal/Network/NimbusClient.cs

```diff
@@ -39,6 +40,12 @@ public NimbusClient(CancellationTokenSource ctx, NimbusSDKConfiguration configur
 			_ctx = ctx;
 		}
 
+		public void AddHeader(List<KeyValuePair<string, string>> headers) {
```

Comment:
This does not seem to be used anywhere, is there a file missing from this PR?

Reply:
Whoops, test code.